x-pack/plugins/security_solution/public/detections/components/alerts_table/timeline_actions/alert_context_menu.tsx

```diff
@@ -60,9 +60,6 @@ const AlertContextMenuComponent: React.FC<AlertContextMenuProps> = ({
   const dispatch = useDispatch();
   const [, dispatchToaster] = useStateToaster();
   const [isPopoverOpen, setPopover] = useState(false);
-  const [alertStatus, setAlertStatus] = useState<Status | undefined>(
```

Comment:
This useState was causing the render bug, and since we aren't actually utilizing the `setAlertStatus` to do anything functional outside this component, I've changed it to always getting the alert status directly from the ECS data as it'll always be correct when rendered.